### PR TITLE
adds css + svg for playground call-out

### PR DESCRIPTION
**Pull request by jpvajda**
- adds the css for the new API Playground callout
- adds codey svg
- tested locally, renders correctly.

Screenshot 2024-05-31 at 4 27 08 PM

### CODE REVIEW

**Comment by jpvajda** [comment]
@SandraRodgers  I could use some help getting the SVG to positioned within the call-out itself based on the design.

Screenshot 2024-05-31 at 4 22 09 PM

File: style.css [hunk 1]
```diff
@@ -550,6 +550,21 @@ Callout styling
   --emoji: unset;
 }
 
+.markdown-body .callout[theme="🛝"] {
+  background-origin: border-box;
+  border: 0.125em solid transparent;
```
Comment (SandraRodgers) [inline style.css:555]:
```suggestion
  border: 0.125em solid transparent;
```

File: style.css [hunk 2]
```diff
@@ -550,6 +550,21 @@ Callout styling
...
...color='%23F185BE'/%3E%3Cstop offset='0.723958' stop-color='%23EE028C'/%3E%3Cstop offset='1' stop-color='%23DD0070'/%3E%3C/linearGradient%3E%3ClinearGradient id='paint11_linear_2055_2802' x1='47.1588' y1='269.367' x2='242.599' y2='465.048' gradientUnits='userSpaceOnUse'%3E%3Cstop stop-color='%23F185BE'/%3E%3Cstop offset='0.479167' stop-color='%23EE028C'/%3E%3Cstop offset='1' stop-color='%23DD0070'/%3E%3C/linearGradient%3E%3ClinearGradient id='paint12_linear_2055_2802' x1='-86.5141' y1='242.506' x2='926.838' y2='549.537' gradientUnits='userSpaceOnUse'%3E%3Cstop offset='0.0961853' stop-color='%23F9B6D9'/%3E%3Cstop offset='0.336076' stop-color='%23F185BE'/%3E%3Cstop offset='0.714691' stop-color='%23EE028C'/%3E%3Cstop offset='0.957356' stop-color='%23DD0070'/%3E%3C/linearGradient%3E%3ClinearGradient id='paint13_linear_2055_2802' x1='824.999' y1='398' x2='1091.36' y2='561.228' gradientUnits='userSpaceOnUse'%3E%3Cstop stop-color='%23DD0070'/%3E%3Cstop offset='0.366379' stop-color='%23EE028C'/%3E%3Cstop offset='0.764637' stop-color='%23F185BE'/%3E%3C/linearGradient%3E%3CclipPath id='clip0_2055_2802'%3E%3Crect width='1016' height='753' fill='white'/%3E%3C/clipPath%3E%3C/defs%3E%3C/svg%3E');
```
Comment (SandraRodgers) [inline style.css:563]:
```suggestion
--icon: url('https://res.cloudinary.com/deepgram/image/upload/v1717523786/devex/codey-greeting_rekqe1.svg');
```
```suggestion
--icon: url('data:image/svg+xml,%3Csvg width='1016' height='753' viewBox='0 0 1016 753' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cg clip-path='url(%23clip0_2055_2802)'%3E%3Cpath d='M442.803 23.3426C430.43 9.12785 412.252 0 391.979 0H349.769C312.793 0 282.527 30.236 282.527 67.1753V105.446C282.527 142.385 312.793 172.621 349.769 172.621H391.979C403.591 172.621 414.536 169.626 424.101 164.396C418.105 197.009 402.639 238.418 370.184 289.999L417.248 319.617C544.879 116.998 447.086 27.0983 442.851 23.3901L442.803 23.3426Z' fill='url(%23paint0_linear_2055_2802)'/%3E%3Cpath d='M690.928 23.3426C678.555 9.12785 660.377 0 640.104 0H597.894C560.918 0 530.652 30.236 530.652 67.1753V105.446C530.652 142.385 560.918 172.621 597.894 172.621H640.104C651.716 172.621 662.661 169.626 672.226 164.396C666.23 197.009 650.764 238.418 618.309 289.999L665.373 319.617C793.004 116.998 695.211 27.0983 690.976 23.3901L690.928 23.3426Z' fill='url(%23paint1_linear_2055_2802)'/%3E%3Cpath d='M337.875 753C328.547 753 320.457 746.011 319.411 736.503C300.137 568.209 328.643 519.859 331.926 514.868L362.906 535.5L363.239 535.025C363.001 535.453 338.874 579.095 356.386 732.272C357.576 742.493 350.2 751.716 340.016 752.857C339.302 752.952 338.588 753 337.875 753Z' fill='url(%23paint2_linear_2055_2802)'/%3E%3Cpath d='M276.007 753C266.68 753 258.59 746.011 257.543 736.503C238.27 568.209 266.775 519.859 270.059 514.868L301.039 535.5L301.372 535.025C301.134 535.453 277.007 579.095 294.519 732.272C295.709 742.493 288.333 751.716 278.149 752.857C277.435 752.952 276.721 753 276.007 753Z' fill='url(%23paint3_linear_2055_2802)'/%3E%3Cpath d='M399.738 753C390.411 753 382.321 746.011 381.274 736.503C362.001 568.209 390.506 519.859 393.789 514.868L424.769 535.5L425.102 535.025C424.864 535.453 400.737 579.095 418.25 732.272C419.439 742.493 412.063 751.716 401.879 752.857C401.165 752.952 400.452 753 399.738 753Z' fill='url(%23paint4_linear_2055_2802)'/%3E%3Cpath d='M461.601 753C452.274 753 444.184 746.011 443.137 736.503C423.864 568.209 452.369 519.859 455.653 514.868L486.632 535.5L486.965 535.025C486.728 535.453 462.6 579.095 480.113 732.272C481.303 742.493 473.926 751.716 463.743 752.857C463.029 752.952 462.315 753 461.601 753Z' fill='url(%23paint5_linear_2055_2802)'/%3E%3Cpath d='M707.298 753C706.584 753 705.87 753 705.156 752.857C694.925 751.669 687.597 742.446 688.786 732.272C706.299 579.095 682.219 535.453 681.934 535.025L682.267 535.5L713.246 514.868C716.578 519.812 745.035 568.161 725.762 736.503C724.667 746.011 716.625 753 707.298 753Z' fill='url(%23paint6_linear_2055_2802)'/%3E%3Cpath d='M769.161 753C768.447 753 767.734 753 767.02 752.857C756.788 751.669 749.46 742.446 750.65 732.272C768.162 579.095 744.082 535.453 743.797 535.025L744.13 535.5L775.11 514.868C778.441 519.812 806.898 568.161 787.625 736.503C786.531 746.011 778.488 753 769.161 753Z' fill='url(%23paint7_linear_2055_2802)'/%3E%3Cpath d='M645.435 753C644.721 753 644.007 753 643.293 752.857C633.062 751.669 625.733 742.446 626.923 732.272C644.435 579.095 620.356 535.453 620.07 535.025L620.403 535.5L651.383 514.868C654.714 519.812 683.172 568.161 663.899 736.503C662.804 746.011 654.762 753 645.435 753Z' fill='url(%23paint8_linear_2055_2802)'/%3E%3Cpath d='M583.571 753C583.333 753 583.095 753 582.81 753C572.531 752.62 564.536 743.967 564.917 733.698C571.103 576.386 556.636 531.935 556.494 531.507C552.877 521.904 557.826 511.302 567.439 507.689C577.099 504.076 587.854 509.115 591.471 518.718C593.231 523.473 608.65 569.777 602.178 735.172C601.797 745.203 593.517 753.047 583.571 753.047V753Z' fill='url(%23paint9_linear_2055_2802)'/%3E%3Cpath d='M523.468 753C513.665 753 505.479 745.394 504.908 735.505C495.343 573.343 504.813 523.615 505.242 521.571C507.24 511.492 517.043 504.979 527.18 506.976C537.221 508.973 543.788 518.719 541.789 528.797C541.599 529.701 532.985 578.62 542.122 733.318C542.741 743.587 534.889 752.382 524.61 753C524.229 753 523.848 753 523.515 753H523.468Z' fill='url(%23paint10_linear_2055_2802)'/%3E%3Cpath d='M178.406 448.786C110.498 448.786 55.2488 415.697 5.85261 347.381C-4.14083 333.547 -1.04762 314.245 12.8004 304.214C26.6485 294.23 45.9691 297.321 56.0102 311.155C118.255 397.299 177.121 398.583 251.882 372.72C268.014 367.158 285.621 375.668 291.237 391.832C296.805 407.948 288.286 425.538 272.106 431.148C238.224 442.843 207.197 448.786 178.406 448.786Z' fill='url(%23paint11_linear_2055_2802)'/%3E%3Cpath d='M927.297 421.212V421.165C927.297 326.748 850.681 250.208 756.171 250.208L298.329 250.208C203.819 250.208 127.203 326.748 127.203 421.165V421.212C127.203 515.629 203.819 592.169 298.329 592.169H756.171C850.681 592.169 927.297 515.629 927.297 421.212Z' fill='url(%23paint12_linear_2055_2802)'/%3E%3Cpath d='M370.806 148.898C405.419 148.898 433.479 120.866 433.479 86.2866C433.479 51.7073 405.419 23.6753 370.806 23.6753C336.193 23.6753 308.133 51.7073 308.133 86.2866C308.133 120.866 336.193 148.898 370.806 148.898Z' fill='white'/%3E%3Cpath d='M378.417 84.0523C377.608 81.1523 374.943 78.5376 371.754 78.2999C368.994 78.1097 366.139 79.1081 364.426 81.39C363.331 82.8163 362.855 84.5277 362.855 86.3343C362.855 87.9507 363.331 89.8998 364.473 91.1359C364.569 91.231 364.664 91.3736 364.759 91.4687C366.044 92.8949 367.709 94.131 369.756 94.2736C371.85 94.4162 373.563 93.8933 375.276 92.7523C375.419 92.6572 375.514 92.5621 375.657 92.5146C376.132 92.0392 376.608 91.5638 377.084 91.0884C377.417 90.4703 377.75 89.8999 378.131 89.2818C378.893 87.6654 378.845 85.7163 378.369 84.0999L378.417 84.0523Z' fill='%23231F20'/%3E%3Cpath d='M618.974 148.898C653.587 148.898 681.647 120.866 681.647 86.2866C681.647 51.7073 653.587 23.6753 618.974 23.6753C584.361 23.6753 556.301 51.7073 556.301 86.2866C556.301 120.866 584.361 148.898 618.974 148.898Z' fill='white'/%3E%3Cpath d='M626.542 84.0523C625.733 81.1523 623.068 78.5376 619.879 78.2999C617.119 78.1097 614.264 79.1081 612.551 81.39C611.456 82.8163 610.98 84.5277 610.98 86.3343C610.98 87.9507 611.456 89.8998 612.598 91.1359C612.694 91.231 612.789 91.3736 612.884 91.4687C614.169 92.8949 615.834 94.131 617.881 94.2736C619.975 94.4162 621.688 93.8933 623.401 92.7523C623.544 92.6572 623.639 92.5621 623.782 92.5146C624.257 92.0392 624.733 91.5638 625.209 91.0884C625.542 90.4703 625.875 89.8999 626.256 89.2818C627.018 87.6654 626.97 85.7163 626.494 84.0999L626.542 84.0523Z' fill='%23231F20'/%3E%3Cpath d='M505.144 344.244C509.142 344.244 512.33 347.762 511.807 351.708C508.142 379.424 484.396 400.865 455.605 400.865C426.815 400.865 403.068 379.471 399.404 351.708C398.881 347.762 402.069 344.244 406.066 344.244H505.097H505.144Z' fill='black'/%3E%3Cpath d='M445.66 390.881C445.66 390.881 463.458 355.368 499.006 364.258C499.006 364.258 490.107 390.881 445.66 390.881Z' fill='white'/%3E%3Cpath d='M985.067 561.077C972.123 561.077 960.036 552.9 955.705 539.922C924.154 445.363 866.573 426.442 814.464 420.404C797.476 418.455 785.341 403.099 787.292 386.127C789.243 369.155 804.614 356.984 821.603 358.981C920.252 370.391 981.498 421.687 1014.43 520.382C1019.85 536.594 1011.05 554.089 994.87 559.461C991.634 560.554 988.303 561.077 985.067 561.077Z' fill='url(%23paint13_linear_2055_2802)'/%3E%3C/g%3E%3Cdefs%3E%3ClinearGradient id='paint0_linear_2055_2802' x1='309.034' y1='107.395' x2='614.615' y2='262.431' gradientUnits='userSpaceOnUse'%3E%3Cstop stop-color='%23F185BE'/%3E%3Cstop offset='0.630208' stop-color='%23EE028C'/%3E%3Cstop offset='1' stop-color='%23DD0070'/%3E%3C/linearGradient%3E%3ClinearGradient id='paint1_linear_2055_2802' x1='504.194' y1='109.487' x2='774.631' y2='198.07' gradientUnits='userSpaceOnUse'%3E%3Cstop stop-color='%23F185BE'/%3E%3Cstop offset='0.635417' stop-color='%23EE028C'/%3E%3Cstop offset='1' stop-color='%23DD0070'/%3E%3C/linearGradient%3E%3ClinearGradient id='paint2_linear_2055_2802' x1='278.104' y1='668.139' x2='399.584' y2='597.913' gradientUnits='userSpaceOnUse'%3E%3Cstop stop-color='%23EDEDF2'/%3E%3Cstop offset='0.192708' stop-color='%23F185BE'/%3E%3Cstop offset='0.572917' stop-color='%23EE028C'/%3E%3Cstop offset='1' stop-color='%23DD0070'/%3E%3C/linearGradient%3E%3ClinearGradient id='paint3_linear_2055_2802' x1='216.237' y1='668.139' x2='337.716' y2='597.913' gradientUnits='userSpaceOnUse'%3E%3Cstop stop-color='%23EDEDF2'/%3E%3Cstop offset='0.192708' stop-color='%23F185BE'/%3E%3Cstop offset='0.572917' stop-color='%23EE028C'/%3E%3Cstop offset='1' stop-color='%23DD0070'/%3E%3C/linearGradient%3E%3ClinearGradient id='paint4_linear_2055_2802' x1='339.968' y1='668.139' x2='461.447' y2='597.913' gradientUnits='userSpaceOnUse'%3E%3Cstop stop-color='%23EDEDF2'/%3E%3Cstop offset='0.192708' stop-color='%23F185BE'/%3E%3Cstop offset='0.572917' stop-color='%23EE028C'/%3E%3Cstop offset='1' stop-color='%23DD0070'/%3E%3C/linearGradient%3E%3ClinearGradient id='paint5_linear_2055_2802' x1='401.831' y1='668.139' x2='523.31' y2='597.913' gradientUnits='userSpaceOnUse'%3E%3Cstop stop-color='%23EDEDF2'/%3E%3Cstop offset='0.192708' stop-color='%23F185BE'/%3E%3Cstop offset='0.572917' stop-color='%23EE028C'/%3E%3Cstop offset='1' stop-color='%23DD0070'/%3E%3C/linearGradient%3E%3ClinearGradient id='paint6_linear_2055_2802' x1='767.068' y1='668.139' x2='645.589' y2='597.913' gradientUnits='userSpaceOnUse'%3E%3Cstop stop-color='%23EDEDF2'/%3E%3Cstop offset='0.182292' stop-color='%23F185BE'/%3E%3Cstop offset='0.598958' stop-color='%23EE028C'/%3E%3Cstop offset='1' stop-color='%23DD0070'/%3E%3C/linearGradient%3E%3ClinearGradient id='paint7_linear_2055_2802' x1='828.931' y1='668.139' x2='707.452' y2='597.913' gradientUnits='userSpaceOnUse'%3E%3Cstop stop-color='%23EDEDF2'/%3E%3Cstop offset='0.182292' stop-color='%23F185BE'/%3E%3Cstop offset='0.598958' stop-color='%23EE028C'/%3E%3Cstop offset='1' stop-color='%23DD0070'/%3E%3C/linearGradient%3E%3ClinearGradient id='paint8_linear_2055_2802' x1='705.205' y1='668.139' x2='583.726' y2='597.913' gradientUnits='userSpaceOnUse'%3E%3Cstop stop-color='%23EDEDF2'/%3E%3Cstop offset='0.182292' stop-color='%23F185BE'/%3E%3Cstop offset='0.598958' stop-color='%23EE028C'/%3E%3Cstop offset='1' stop-color='%23DD0070'/%3E%3C/linearGradient%3E%3ClinearGradient id='paint9_linear_2055_2802' x1='643.865' y1='667.284' x2='513.729' y2='592.058' gradientUnits='userSpaceOnUse'%3E%3Cstop stop-color='%23EDEDF2'/%3E%3Cstop offset='0.317708' stop-color='%23F185BE'/%3E%3Cstop offset='0.739583' stop-color='%23EE028C'/%3E%3Cstop offset='1' stop-color='%23DD0070'/%3E%3C/linearGradient%3E%3ClinearGradient id='paint10_linear_2055_2802' x1='461.984' y1='665.24' x2='584.89' y2='594.251' gradientUnits='userSpaceOnUse'%3E%3Cstop stop-color='%23EDEDF2'/%3E%3Cstop offset='0.265625' stop-color='%23F185BE'/%3E%3Cstop offset='0.723958' stop-color='%23EE028C'/%3E%3Cstop offset='1' stop-color='%23DD0070'/%3E%3C/linearGradient%3E%3ClinearGradient id='paint11_linear_2055_2802' x1='47.1588' y1='269.367' x2='242.599' y2='465.048' gradientUnits='userSpaceOnUse'%3E%3Cstop stop-color='%23F185BE'/%3E%3Cstop offset='0.479167' stop-color='%23EE028C'/%3E%3Cstop offset='1' stop-color='%23DD0070'/%3E%3C/linearGradient%3E%3ClinearGradient id='paint12_linear_2055_2802' x1='-86.5141' y1='242.506' x2='926.838' y2='549.537' gradientUnits='userSpaceOnUse'%3E%3Cstop offset='0.0961853' stop-color='%23F9B6D9'/%3E%3Cstop offset='0.336076' stop-color='%23F185BE'/%3E%3Cstop offset='0.714691' stop-color='%23EE028C'/%3E%3Cstop offset='0.957356' stop-color='%23DD0070'/%3E%3C/linearGradient%3E%3ClinearGradient id='paint13_linear_2055_2802' x1='824.999' y1='398' x2='1091.36' y2='561.228' gradientUnits='userSpaceOnUse'%3E%3Cstop stop-color='%23DD0070'/%3E%3Cstop offset='0.366379' stop-color='%23EE028C'/%3E%3Cstop offset='0.764637' stop-color='%23F185BE'/%3E%3C/linearGradient%3E%3CclipPath id='clip0_2055_2802'%3E%3Crect width='1016' height='753' fill='white'/%3E%3C/clipPath%3E%3C/defs%3E%3C/svg%3E');
```

File: style.css [hunk 3]
```diff
@@ -550,6 +550,21 @@ Callout styling
...
+    #ff2eea -99.24%,
+    #4b3cff 99.26%
+  ) !important;
+  box-shadow: 0.125em 1000px 1px #101014 inset;
+
+--icon: url('https://res.cloudinary.com/deepgram/image/upload/v1717523786/devex/codey-greeting_rekqe1.svg');
+
+  --emoji: unset;
+}
+
```
Comment (SandraRodgers) [inline style.css:567]:
```suggestion

.callout.callout .callout-heading.empty > .callout-icon{
		float: left;
    margin-top: .5rem;
    margin-right: 1.75rem;
  	margin-left: .15rem;
    transform: scaleX(-1);
}
```

**Review by SandraRodgers — APPROVED** [review]

**Pull request merged**